tests/Fixtures/Integration/misc/phpdocs-invalid.test

```diff
@@ -0,0 +1,40 @@
+--TEST--
+Invalid phpdocs must be kept unfixed.
+--RULESET--
+{
+    "@PhpCsFixer": true,
+    "@PhpCsFixer:risky": true
+}
+--EXPECT--
+<?php
+
+/**
+ * @method X|Y testA(int $x)
+ * @method X   testB(int $$x)
+ * @method X|Y testC(int $x)
+ */
+class Foo
+{
+    /** @var (X || Y) */
+    public $v1;
+
+    /** @var (X|Y) */
+    public $v2;
+}
+
+--INPUT--
+<?php
+
+/**
+ * @method  Y|X testA(int $x)
+ * @method  X testB(int $$x)
+ * @method  Y|X testC(int $x)
+ */
+class Foo
+{
+    /** @var  (X || Y) */
+    public $v1;
+
+    /** @var  (X | Y) */
```

Comment:
For consistency with `@method`s above and to see if `$v2`'s types are sorted:

```suggestion
    /** @var  (Y || X) */
    public $v1;

    /** @var  (Y | X) */
```

Reply:
This is a test for invalid phpdoc. `X | Y` is valid, `X || Y` is invalid.

Reply:
I know, but it does not change my point - there's `@method  Y|X testA(int $x)` above which is sorted (expected output has `@method X|Y testA(int $x)`. Shouldn't `@var  (Y | X)` be sorted the same way, hence proposed inverted order in the input?

Reply:
The test is primary there to assert invalid phpdocs stay as is and fixer can finish, but I have improved the test 👍